sigmf-schema.json

```diff
@@ -76,7 +91,7 @@
           "default": 0,
           "minimum": 0,
           "!comment": "The maximum value for this property is equal to 2^63 - 1, making it easy to fit into a signed 64-bit integer.",
-          "maximum": 9223372036854775807,
+          "maximum": 9223372036854776000,
```

Comment:
I don't think you meant to change these. This number is `2**63 + 192`. Was previously max signed integer.